### PR TITLE
Deal with NaN's in y

### PR DESCRIPTION
**Pull request by lyon-fnal**
This PR fixes issue #149

### CODE REVIEW

File: src/bar.jl [hunk 1]
```diff
@@ -48,6 +48,7 @@ grouped_xy(y::AbstractMatrix) = 1:size(y,1), y
     fillrange := if isstack
         # shift y/fillrange up
         y = copy(y)
+        y[isnan.(y)] = 0
```
Comment (mkborregaard) [inline src/bar.jl]: Perhaps better `!isfinite.(y)`?

Comment (lyon-fnal) [reply src/bar.jl]: That works, though the deprecation warning said to do `y[.!isfinite.(y)] = 0`. I'll update the PR.

**Comment by mkborregaard** [comment]
Thanks a lot @lyon-fnal !

**Comment by lyon-fnal** [comment]
Happy to help!

**Pull request merged**